### PR TITLE
Update account documentation for Stable Cadence

### PR DESCRIPTION
**Pull request by turbolent**
- Remove capability controllers from current docs
- Update account documentation to `Account` type and entitlements

In a next PR I'm planning to split the Accounts page into sub-pages. The capability API is already a separate page / split from the rest of the Account API

### CODE REVIEW

File: docs/cadence/language/accounts.mdx [hunk 1]
```diff
@@ -469,44 +400,202 @@ access(all) struct AuthAccount {
...
 ```
 
-A script can get the `AuthAccount` for an account address using the built-in `getAuthAccount` function:
+### Account access
+
+#### Performing read operations
+
+Access to an `&Account` means having "read access" to it.
+For example, the `address` and `balance` fields have the `access(all)` modifier,
+so are always accessibly, which is safe because this information is public,
```
Comment (dsainati1) [inline docs/cadence/language/accounts.mdx:502]:
```suggestion
so are always accessible, which is safe because this information is public,
```

**Review by dsainati1 — COMMENTED** [review]
Does this need some kind of explanation of how the entitlement mappings for accounts propagate access? I.e. something indicating that accessing the `keys` sub-object with the coarse grained entitlement would give you all the fine-grained entitlements, for example.

**Comment by turbolent** [comment]
> Does this need some kind of explanation of how the entitlement mappings for accounts propagate access? I.e. something indicating that accessing the `keys` sub-object with the coarse grained entitlement would give you all the fine-grained entitlements, for example.

Good idea! I'll add an explanation in a follow up. It would be good to link to the documentation for the entitlements / entitlement mappings language feature, do we have that already somewhere?

**Comment by dsainati1** [comment]
> It would be good to link to the documentation for the entitlements / entitlement mappings language feature, do we have that already somewhere

Yeah it's here: https://developers.flow.com/next/cadence/language/access-control#entitlement-mappings

**Review by bthaile — APPROVED** [review]
lgtm

File: docs/cadence/language/accounts.mdx [hunk 2]
```diff
@@ -3,284 +3,184 @@ title: Accounts
...
+Accounts are only accessed through [references](./references.mdx),
+which might be [authorized](./references.mdx#authorized-references).
 
-## `PublicAccount`
-
-**Public Account** objects have the type `PublicAccount`,
-which represents the publicly available portion of an account.
+Account objects provide information about and allow the management of
+for example the account's [storage](#account-storage), [keys](#account-keys),
+and [capabilities](./capability-based-access-control.md).
```
Comment (SupunS) [inline docs/cadence/language/accounts.mdx:12]:
```suggestion
Account objects provide information about and allow the management of
different aspects of the account such as the account's [storage](#account-storage), [keys](#account-keys),
, [capabilities](./capability-based-access-control.md), and etc.
```

File: docs/cadence/language/accounts.mdx [hunk 3]
```diff
@@ -469,44 +400,202 @@ access(all) struct AuthAccount {
...
+
+#### Performing read operations
+
+Access to an `&Account` means having "read access" to it.
+For example, the `address` and `balance` fields have the `access(all)` modifier,
+so are always accessible, which is safe because this information is public,
+and the fields are read-only.
+
+Any code can get a "read-only" reference to an account (`&Account`)
+for an account address by using the built-in `getAccount` function:
```
Comment (SupunS) [inline docs/cadence/language/accounts.mdx:506]:
```suggestion
Any code can get a "read-only" reference to an account (`&Account`)
at a given address by using the built-in `getAccount` function:
```

File: docs/cadence/language/accounts.mdx [hunk 4]
```diff
@@ -469,44 +400,202 @@ access(all) struct AuthAccount {
...
+Though scripts can perform write operations,
+they discard their changes upon completion.
+Attempting to use this function outside of a script,
+for example in a transaction,
+causes a type error.
+
+## Creating an account
+
+The `Account` constructor allows creating new accounts.
+The function requires a reference to a _payer_ account,
```
Comment (SupunS) [inline docs/cadence/language/accounts.mdx:568]: Maybe also mention the payer account reference needs (Storage | BorrowValue) entitlement

**Review by SupunS — APPROVED** [review]

**Review by dsainati1 — APPROVED** [review]

File: docs/cadence/language/accounts.mdx (hunk 4 above)
Comment (turbolent) [reply docs/cadence/language/accounts.mdx]: Will do that in a follow-up to avoid conflicts

**Pull request merged**